app/helpers/blacklight/facets_helper_behavior.rb

```diff
@@ -169,7 +169,8 @@ def facet_partial_name(display_facet = nil)
   deprecation_deprecate :facet_partial_name
 
   def facet_field_presenter(facet_config, display_facet)
-    Blacklight::FacetFieldPresenter.new(facet_config, display_facet, self)
+    presenter_class = facet_config.presenter || Blacklight::FacetFieldPresenter
```

Comment:
could we set this default in the facet_config itself to avoid the conditional here?

Reply:
🤷‍♂️ , sure.